src/sentry/uptime/endpoints/uptime_ips.py

```diff
@@ -4,11 +4,10 @@
 from sentry import options
 from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
-from sentry.api.base import Endpoint, all_silo_endpoint
+from sentry.api.base import Endpoint, control_silo_endpoint
 
 
-# TODO(cells): This endpoint is moving to control
-@all_silo_endpoint
+@control_silo_endpoint
 class UptimeIpsEndpoint(Endpoint):
```

Comment:
@evanpurkhiser where is this endpoint being used? it doesn't seem to be via our UI. anywhere else we need to update before turning it off on the region API?

Reply:
Yes -- it's accessed via `http://sentry.io/api/0/uptime-ips/` only, and not `http://us.sentry.io/api/0/uptime-ips/` and `http://de.sentry.io/api/0/uptime-ips/`

Reply:
reverted this part for now, we'll wait a "few months"